srsue/src/stack/rrc/rrc.cc

```diff
@@ -1718,6 +1718,13 @@ void rrc::send_ul_dcch_msg(uint32_t lcid, const ul_dcch_msg_s& msg)
     }
   }
 
+  // FUZZING HERE
+  for(int i = 0; i < 4; ++i){
```

Comment:
Instead of a loop to 4, shouldn't this be to the number of bits to fuzz? Likewise, avoid an `int`, use a `uint32_t`.

Reply:
I updated LTE to be just like NR, I just hadn't had the chance to mess with it.